packages/components/src/InputEmail/InputEmail.rebuilt.tsx

```diff
@@ -48,33 +47,15 @@ export const InputEmailRebuilt = forwardRef(function InputEmailInternal(
       inputRef,
     });
 
-  const inputProps = omit(props, [
-    "placeholder",
-    "onChange",
-    "onBlur",
-    "onFocus",
-    "onEnter",
-    "size",
-    "prefix",
-    "suffix",
-    "version",
-  ]);
-
-  const { fieldProps, descriptionIdentifier } = useInputEmailFormField({
-    ...inputProps,
-    id,
-    name,
-    handleChange,
-    handleBlur,
-    handleFocus,
-    handleKeyDown,
-  });
+  const descriptionIdentifier = `descriptionUUID--${id}`;
+  const hasDescription = props.description && !props.inline;
+  const isInvalid = Boolean(props.ariaInvalid || props.error || props.invalid);
```

Comment:
this makes me wonder if ariaInvalid belongs as an exposed prop

it doesn't really make sense to only pass `ariaInvalid` - there are specific invalid states that we use. you'd never only pass in this prop, it should always be derived from one of the others.

yeah I don't think this makes sense to offer on our inputs. I'll remove it.